File: Lista_Geral_C/comandos de entrada e saida/question24.c
// Escreva um programa que leia o termo inicial e a razão de uma progressão aritmética e
// imprima os 5 primeiros termos desta progressão.
#include <stdio.h>

int main(){

    int razao, termo_inicial;

    printf("Informe o termo incial da PA: ");
    scanf("%d", &termo_inicial);
    printf("Informe a razao da PA: ");
    scanf("%d", &razao);

    printf("O 1 termo e: %d\nO 2 termo e: %d\nO 3 termo e: %d\nO 4 termo e: %d\nO 5 termo e: %d",termo_inicial, termo_inicial +razao, termo_inicial + (razao*2), termo_inicial+razao*3, termo_inicial+razao*4);

    return 0;
}

File: Lista_Geral_Python/comandos de entrada e saida/question24.py
### Escreva um programa que leia o termo inicial e a razão de uma progressão aritmética e
# imprima os 5 primeiros termos desta progressão.
termo_inicial = int(input("Informe o termo incial da PA: "))
razao = int(input("Informe a razão da PA: "))
print(f"""O 1º termo é: {termo_inicial}
O 2º termo é: {termo_inicial + razao}
O 3º termo é: {termo_inicial + razao * 2}
O 4º termo é: {termo_inicial + razao * 3}
O 5º termo é: {termo_inicial + razao * 4}""")